easybuild/easyconfigs/j/JasPer/JasPer-1.900.1-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,47 @@
+# Built with EasyBuild version 3.3.1 on 2017-12-13_16-42-36
```

Comment:
JasPer 2.0.12 is already installed on daint (as hidden dependency), you could try that (and remove this eb from the PR if it works).